### PR TITLE
Update README to include test details and json collection

### PR DESCRIPTION
**Pull request by conormiley**
@richardadamdean

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+This is a collection of a simple set of publicly available resources for engineers submitting technical tests to amaysim.
+
+
+## What we're looking for
+Here at amaysim we're all about creating beautiful, enriching and rewarding experiences for our customers. We value simplicity and reliability in how we develop our software. With that in mind we'd love for you to build a very simple app for us that our Product manager wants to get up and running for some User testing.
+
+
+## Technical Test
+
+Using our public `collection.json` as a source of customer data we want you to create an app that consumes data from the collection and does the following:
```
Comment (richardadamdean) [inline README.md]: Maybe mention this is json-api (and link to http://jsonapi.org/)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
+
+1. Create an initial login screen that allows a customer to login (we don't want you to actually create an authentication layer)
```
Comment (richardadamdean) [inline README.md]: Mention that they should just validate the credentials in the json

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+  * information about their plan, also known as a subscription,
+remaining data balance of their subscription
+  * information about the product that is related to the subscription, including the name and price of the product
+  * any other information you think may be relevant to the customer
+
+## Technical Guidance
+
+1. Use the programming language and tooling of your choice, either Objective C/Swift/Java
+2. Make your source code available on Github/Bitbucket (or the online repo of your choosing)
+3. Try to avoid the use of frameworks, unless their for testing
```
Comment (richardadamdean) [inline README.md]: Maybe this is less relevant for swift?

File: README.md (hunk 1 above)
Comment (conormiley) [inline README.md]: done and link added to resource

**Pull request merged**